### PR TITLE
Global hotkeys to toggle visibility + completely hide the GUI

### PR DESCRIPTION
**Pull request by timmolderez**
Hi José,

This pull request adds support for two global hotkeys:
- Win+Shift+F6 : same effect as clicking the "Toggle visibility" button
- Win+Shift+F7 : toggles "stealth mode"; this completely hides the GUI, so it won't even show up e.g. when Alt+Tabbing between windows

So people can discover the existence of these two hotkeys:
- Added a tooltip to the Toggle visibility button, mentioning the Win+Shift+F6 hotkey
- Added a Minimize button to the GUI window; using it will actually trigger stealth mode + show a message mentioning Win+Shift+F7

Maybe useful as well to include a few words on how I use Win11ClockToggler: in Windows 10, I always keep the clock hidden + set the taskbar size to small, for a superclean/no-distractions look. As you know, can't do neither of those in Win 11 anymore; boo! 

I tried just about every utility to bring back my clean look in Win 11, but yours works the best! :) (in combination with changing the registry to make the taskbar small again) This means I leave the Win11ClockToggler GUI running all the time; all I needed to make it perfect were these two hotkeys to easily toggle the notification area, and hide the GUI window when I don't need it. Anyway, just wanted to share in case it might be useful for someone else.

In short, thanks so much for developing this excellent utility, finally making Win 11 enjoyable to use :)

Cheers,
 Tim

### CODE REVIEW

**Comment by jmalarcon** [comment]
Hey Tim, thank you very much for your addition to the app!

I think it's very useful. I normally use my app only to do screen recordings, and if I needed to use ALT+TAB simply sent the app window to a secondary desktop (or to the secondary screen, with app buttons only showing in the screen they are). But this is far better. Thanks!

I'm accepting your PR and will change the version number to release a new version.

Maybe in the future it would be useful to add a settings screen to be able to select custom keys to do this. Anyway I'd like to keep the app simple, so no need for that right now.

Thanks again!

**Comment by timmolderez** [comment]
That's quick; thanks! 😀 Indeed; also considered making the hotkeys customizable, but imagine it would add substantial code to make it user-friendly. (Ah, this looks like a useful starting point though: https://stackoverflow.com/questions/55247885/configurable-hotkeys-for-button-presses-c-sharp )
